apps/cli/package.json

```
@@ -6,11 +6,15 @@
   },
   "scripts": {
     "prepare": "yarn build",
+    "lint": "tsc",
+    "test": "jest src",
```

Is this position argument needed here? 🤔 
```suggestion
    "test": "jest",
```